common/lib/kontena-logging.rb

```diff
@@ -0,0 +1,3 @@
+module Kontena
+  require_relative './kontena/logging'
```

Comment:
This already exists as `lib/kontena/logging.rb` and can be required by `require 'kontena/logging'`. I don't think this `kontena-logging.rb` adds much value.